README.md

```diff
@@ -29,9 +27,9 @@ Shows all of the XAML controls in an interactive format. This app is the interac
 
 ## Further information
 
->**Note**: The WinUI 3 Xaml Controls Gallery requires [Visual Studio 16.9 Preview 1](https://visualstudio.microsoft.com/vs/preview/) or later to build and Windows 10 to execute.  Also ensure that you've completed the prerequisite steps from the [installation instructions for WinUI 3 Preview 4](https://aka.ms/winui3/preview4).
+>**Note**: The WinUI 3 Gallery requires [Visual Studio 16.9 Preview 1](https://visualstudio.microsoft.com/vs/preview/) or later to build and Windows 10 to execute.  Also ensure that you've completed the prerequisite steps from the [installation instructions for WinUI 3 Preview 4](https://aka.ms/winui3/preview4).

```

Comment:
Should this be about the WinUI 2 Gallery then?

Reply:
No, it is still WinUI3, but it is the older version of the WinAppSDK.  It is in a separate branch since there are features that were introduced after 1.0.

Reply:
Of course this is not targeting WinUI 2 and you are right, no idea why I thought that in the first place. Sorry about the confusion.